xendit-android/src/main/java/com/xendit/Xendit.java

```diff
@@ -709,6 +709,20 @@ public void onFailure(NetworkError error) {
         }));
     }
 
+    public void unregisterAllBroadcastReceiver() {
+        if (authenticationBroadcastReceiver != null) {
+            context.unregisterReceiver(authenticationBroadcastReceiver);
+        }
+
+        if (tokenBroadcastReceiver != null) {
+            context.unregisterReceiver(tokenBroadcastReceiver);
+        }
+
+        if (authenticatedTokenBroadcastReceiver != null) {
+            context.unregisterReceiver(authenticatedTokenBroadcastReceiver);
+        }
```

Comment:
should make sure to set the variable to null, so it there wont be any unnecessary `unregisterReceiver` when they already been unregistred.